src/file_dialog.rs

```diff
@@ -154,6 +156,8 @@ pub struct FileDialog {
     /// This is used to prevent the dialog from closing when pressing the escape key
     /// inside a text input.
     any_focused_last_frame: bool,
+
+    vfs: Arc<dyn FileSystem + Send + Sync>,
```

Comment:
Could you rename this to `file_system`? We can't know whether it is a virtual file system